tests/test_customreports.py

```diff
@@ -15,31 +17,36 @@ class WarningsNgTestCase(unittest.TestCase):
     def setUpClass(cls) -> None:
         formatter = log.LogMessageFormatter()
         cls.logger = logging.getLogger(__name__)
-        cls.expected_file = "bom.json"
-        cls.expected_target = "issues.json"
+        cls.tempdir = TemporaryDirectory()
+        cls.expected_file = os.path.join(cls.tempdir.name, "bom.json")
+        cls.expected_target = os.path.join(cls.tempdir.name, "issues.json")
         warnings_ng_handler = WarningsNgReporter(
```

Comment:
As you changed the name of the handler in `validate.py`, should it be adjusted to `report_handler`?

Reply:
Just for the sake of consistency